website/docs/getting-started/configure.md

```diff
@@ -4,4 +4,82 @@ sidebar_position: 6
 ---
 # Configuring odo global settings
 
-This doc explains everything in `~/.odo/preference.yaml` file.
\ No newline at end of file
+The global settings for odo can be found in `preference.yaml` file; which is located by default in the `.odo` directory of the user's HOME directory.
+
+Example -
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+defaultValue="Linux"
+values={[
+{label: 'Linux', value: 'linux'},
+{label: 'Windows', value: 'windows'},
+{label: 'Mac', value: 'mac'},
+]}>
+<TabItem value="linux">/home/user/foo/myProject/.odo/preference.yaml</TabItem>
+<TabItem value="windows">C:\\myProject\.odo\preference.yaml</TabItem>
+<TabItem value="mac">/home/user/foo/myProject/.odo/preference.yaml</TabItem>
+</Tabs>
+
+---
+A  different location can be set for the `preference.yaml` by exporting `GLOBALODOCONFIG` in the user environment.
+
+### View the configuration
+To view the current configuration, run `odo preference view`.
+
+```shell
+odo preference view
+```
+Example -
+```shell
+$ odo preference view
+PARAMETER             CURRENT_VALUE
+UpdateNotification
+NamePrefix
+Timeout
+BuildTimeout
+PushTimeout
+Experimental
+Ephemeral
+ConsentTelemetry
+```
+### Set a configuration
+To set a value for a preference key, run `odo preference set <key> <value>`.
+```shell
+odo preference set updatenotification false
+```
+Example -
+```shell
+$ odo preference set updatenotification false
+Global preference was successfully updated
```

Comment:
The output is weird. When `-f` flag is not specified, odo should ask the user if they really want to change the value. Like below:
```sh
$ odo preference set updatenotification false
? updatenotification is already set. Do you want to override it in the config No
Aborted by the user
```

I saw same behaviour as this output (odo not asking me to confirm) but I can't reproduce it any more.

Reply:
So my ask here is to change it to the output I have mentioned in my comment.

Reply:
This seems like a bug to me. `odo preference set` should ask for confirmation.
I think if you unset the value for a key and then set it again, you will be able to reproduce it.

Reply:
> The output is weird. When `-f` flag is not specified, odo should ask the user if they really want to change the value. Like below:
> 
> ```shell
> $ odo preference set updatenotification false
> ? updatenotification is already set. Do you want to override it in the config No
> Aborted by the user
> ```
> 
> I saw same behaviour as this output (odo not asking me to confirm) but I can't reproduce it any more.

This is not the case when no value is set for a key. In that case, my output is correct.